src/vm/eventtracepriv.h

```diff
@@ -291,6 +291,10 @@ class BulkTypeEventLogger
     // List of types we've batched.
     BulkTypeValue m_rgBulkTypeValues[kMaxCountTypeValues];
 
+#ifdef FEATURE_PAL    
+    BYTE m_BulkTypeEventBuffer[65536];
```

Comment:
This is subtle...  The code which prevents too many events does this check.  We should add an assert to make sure the buffer doesn't go above this size, but in practice we never even get close.  If that were the case, we'd see tons of problems on windows too, as it has a 65k buffer limit as well...we just never reach it.